src/xrpld/ledger/detail/View.cpp

```diff
@@ -2838,11 +2838,12 @@ rippleUnlockEscrowMPT(
     ApplyView& view,
     AccountID const& sender,
     AccountID const& receiver,
-    STAmount const& amount,
+    STAmount const& netAmount,
+    STAmount const& grossAmount,
     beast::Journal j)
 {
-    auto const issuer = amount.getIssuer();
-    auto const mptIssue = amount.get<MPTIssue>();
+    auto const issuer = grossAmount.getIssuer();
```

Comment:
Can be reference

Reply:
what do you mean reference? 

`auto const& issuer = grossAmount.getIssuer();`

This? Doesn't auto const already deduce to AccountID const&?